clustering/faces/mojarra/pom.xml

```diff
@@ -54,6 +54,8 @@
         <dependency>
             <groupId>jakarta.faces</groupId>
             <artifactId>jakarta.faces-api</artifactId>
+            <!-- Temporary override until upstream publishes update -->
+            <version>4.0.1</version>
```

Comment:
Why is this needed?

Reply:
With the fix above, it is not. I'll get these both cleaned up.